### DISCUSSION TITLE
"Sign in" and "Content not available" errors plague every video, can't watch anything

### DISCUSSION THREAD

**Discussion by MixusLaxus** [FreeTubeApp/FreeTube#5869] [Q&A]
On the common issues page, it says "if you see this error, feel free to create an issue", so I will, but not before I ask it as a discussion question first in case I'm just missing something.

No matter what video I click on, I get these errors and I can't watch anything, with or without VPN:

1. Invidious API Error: Error: Please sign in
2. Falling back to Local API
3. Local API Error: Error: This content isn't available, try again later.

Or, instead of that last one, I get:
Local API Error: Error: Please sign in

What does it mean "Sign in?" I can't find such settings anywhere, and I thought the whole point of this app was that I didn't need an account...?

This clearly isn't a quota issue because I haven't opened FreeTube in many months, and just updated it.

I tested more popular videos from the biggest channels to see if those would be available, and they weren't either. Strange that Mr. Beast wouldn't be available.

This app hasn't worked for me at all for a very long time, only when I first installed it ages ago did it work for a while. I now want to do something about it because I can't stand the normal YouTube player.

FreeTube Version 0.21.3

MacOS Sequoia 15.0

**Comment by absidue** [comment]
It sounds like you are using a broken Invidious instance and have Proxy Videos Through Invidious enabled, that setting currently applies to both API backends (will only apply to Invidious in the next release). So what is happening is that it is erroring with Invidious, then trying to use the local API which because of your settings also means making a request to the Invidious API and because that fails you get a local API error too, although technically it was caused by Invidious.

The solution is to switch your preferred API backend to the local API and disabled Proxy Videos Through Invidious or find an Invidious instance that works.

**Reply by MixusLaxus** [reply]
That fixed it, thank you!
